conda/activate.py

```diff
@@ -229,6 +229,14 @@ def execute(self):
         context.plugin_manager.invoke_post_commands(self.command)
         return response
 
+    @deprecated(
+        "25.3",
+        "25.9",
+        addendum="Use `conda commands` instead.",
+        # these commands are already pretty hidden in their implementation and access (`conda shell.posix commands`)
+        # so we opt to not warn end users that this is going away, we only need to notify tab-completion devs
+        # deprecation_type=FutureWarning,
```

Comment:
Should this line be uncommented or deleted?

Reply:
technically the comment is commentary on the commented out line